src/dmd/declaration.d

```diff
@@ -996,7 +996,8 @@ extern (C++) final class OverDeclaration : Declaration
                 return true;
             if (auto fd = s.isFuncDeclaration())
             {
-                return fd.isUnique();
+                // isUnique is not callable using a const object
+                return (cast() fd).isUnique();
```

Comment:
What I meant was to make `isUnique` const and do the cast there (in the call to `overloadApply` IIRC).